eng/.runsettings

```diff
@@ -14,5 +14,22 @@
         </Configuration>
       </Logger>
     </Loggers>
+    <Logger friendlyName="blame" enabled="True" />
   </LoggerRunSettings>
-</RunSettings>
\ No newline at end of file
+  <DataCollectionRunSettings>
+    <DataCollectors>
+      <!-- Enables blame -->
+      <DataCollector friendlyName="blame" enabled="True">
+        <Configuration>
+          <!-- Enables crash dump, with dump type "Full" or "Mini".
+          Requires ProcDump in PATH for .NET Framework. -->
```

Comment:
This is kind of a weird comment. We don't do anything for .NET Framework. (I assume this means .NET Framework 4.x)